docs/tools/system-snapshot.md

```
@@ -0,0 +1,148 @@
+    memory: MemoryMetrics | None    # Memory utilization
+    disk: DiskMetrics | None        # Disk I/O rates
+    network: NetworkMetrics | None  # Network I/O rates
+    load: LoadMetrics | None        # Load averages
+    timestamp: str                  # ISO 8601 timestamp
+```
+
+## Example Queries
+
+### Full System Snapshot
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)